x/oracle/types/keys.go

```diff
@@ -55,3 +59,20 @@ func GetAggregateExchangeRateVoteKey(v sdk.ValAddress) (key []byte) {
 	key = append(key, KeyPrefixAggregateExchangeRateVote...)
 	return append(key, address.MustLengthPrefix(v)...)
 }
+
+// GetMedianKey - stored by *denom*
+func GetMedianKey(denom string) (key []byte) {
```

Comment:
```suggestion
func MedianKey(denom string) (key []byte) {
```

Reply:
We are not getting anything from anywhere.

Reply:
While I don't disagree with the naming update the rest of the methods in this file have "Get" in front of them, so updating just these two methods would look inconsistent. I'd suggest doing this in another PR.

Reply:
I agree @rbajollari